src/pages/FundingOpen/GiftSetPage.jsx

```diff
@@ -63,22 +96,25 @@ const GiftSetPage = () => {
             id='gift-image'
             placeholder='+'
             required
```

Comment:
```suggestion
            required={lastGiftData.giftUrl ? false : true}
```
아이템 수정할 때 수정 없이 다음 버튼 누르면 이런 에러가 뜨는 걸 보니까 사진 input이 문제인 듯 합니다
![image](https://github.com/user-attachments/assets/886f2845-7811-46ca-9587-edace34744a7)

required에 이런 식으로 작성해도 되는지 확실하지 않은데 required 값을 무효화 시키면 수정 없이도 다음 버튼 클릭 했을 때 넘어가요!